### PR TITLE
tetragon: Rename killer to enforcer

### PR DESCRIPTION
**Pull request by olsajiri**
rename all the killer stuff to enforcer, plus one simple sensors fix

### CODE REVIEW

File: pkg/sensors/handler.go [hunk 1]
```diff
@@ -251,15 +250,11 @@ func (h *handler) addSensor(op *sensorAdd) error {
...
 }
 
-func removeAllSensors(h *handler) error {
-	var errs error
+func removeAllSensors(h *handler) {
 	for _, col := range h.collections {
-		if err := col.unload(); err != nil {
-			errs = errors.Join(errs, err)
-		}
+		col.destroy()
```
Comment (kkourt) [inline pkg/sensors/handler.go:255]: CC: @mtardy

**Review by kkourt — APPROVED** [review]
Thanks!

File: pkg/sensors/handler.go (hunk 1 above)
Comment (mtardy) [reply pkg/sensors/handler.go:255]: yep indeed this is a miss on my part thanks. The destroy hook was added in contrast to the unload one that was used when `enabling/disabling` vs removing completely the sensor. Allowing us to clean up stuff only when it's really removed and not only disabled.

Comment (kkourt) [reply pkg/sensors/handler.go:255]: Sounds good. Can you please ACK the PR if you agree with the change?

Comment (mtardy) [reply pkg/sensors/handler.go:255]: Too late but yes I agreed with the change

**Comment by lambdanis** [comment]
This PR contains a breaking change to a CRD, so I labeled it as such. @olsajiri could you write in the release note which fields changed?

**Pull request merged**